### PR TITLE
Increment Package Version To v3

### PR DESCRIPTION
**Pull request by SyntaxNode**
Oops. We forgot to do this for v3.0.0.

### CODE REVIEW

File: adapters/criteo/criteo_test.go [hunk 1]
```diff
@@ -2,13 +2,14 @@ package criteo
 
 import (
 	"fmt"
+	"testing"
```
Comment (bsardo) [inline adapters/criteo/criteo_test.go:5]: Noting +2, -1 changed lines here that are unrelated to the version package update.

File: adapters/vidazoo/params_test.go [hunk 2]
```diff
@@ -2,8 +2,9 @@ package vidazoo
 
 import (
 	"encoding/json"
-	"github.com/prebid/prebid-server/v2/openrtb_ext"
 	"testing"
+
```
Comment (bsardo) [inline adapters/vidazoo/params_test.go:6]: Noting +2, -1 changed lines here that are unrelated to the version package update.

File: adapters/yieldlab/types.go [hunk 3]
```diff
@@ -1,9 +1,10 @@
 package yieldlab
 
 import (
-	"github.com/prebid/prebid-server/v2/openrtb_ext"
 	"strconv"
 	"time"
+
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
```
Comment (bsardo) [inline adapters/yieldlab/types.go:7]: Noting +2, -1 changed lines here that are unrelated to the version package update.

File: hooks/hookstage/bidderrequest.go [hunk 4]
```diff
@@ -2,7 +2,8 @@ package hookstage
 
 import (
 	"context"
-	"github.com/prebid/prebid-server/v2/openrtb_ext"
+
```
Comment (bsardo) [inline hooks/hookstage/bidderrequest.go:5]: Noting +1 changed lines here that are unrelated to the version package update.

File: hooks/hookstage/processedauctionrequest.go [hunk 5]
```diff
@@ -2,7 +2,8 @@ package hookstage
 
 import (
 	"context"
-	"github.com/prebid/prebid-server/v2/openrtb_ext"
+
```
Comment (bsardo) [inline hooks/hookstage/processedauctionrequest.go:5]: Noting +1 changed lines here that are unrelated to the version package update.

**Review by bsardo — APPROVED** [review]
LGTM. I went through all of the files quickly in vscode webview.
I also did a search locally on the PR branch for `/v3` including only `*.go, Makefile, Dockerfile` and there are 3310 lines added and 3310 deleted. I pointed out where the additional changes are that bring us to 3320 and 3314 added and deleted respectively for the second reviewer. I successfully started up locally.

**Review by guscarreon — APPROVED** [review]
LGTM

**Comment by SyntaxNode** [comment]
Thanks for the in-depth review. I ran `gofmt` on each file after changing the import path which seems to have caused regrouping / reorganizing in the files @bsardo identified.

**Review by bsardo — APPROVED** [review]

**Review by guscarreon — APPROVED** [review]
LGTM
